### PR TITLE
Categories redirection

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

File: src/Components/Footer/Footer.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .footer {
 	background: var(--bg-color-1) !important;
```
Comment (Dev-AkshatGupta) [inline src/Components/Footer/Footer.css:2]: Why you have given important here didn't got it

File: src/Components/Footer/Footer.css [hunk 2]
```diff
@@ -29,12 +30,13 @@
...
-	font-size: 2rem;
-	text-align: center;
-	border-top: .1rem solid var(--text-color-2);
-	padding: 1rem;
-	padding-top: 2rem;
+	font-size: 1.5rem;
+	/* text-align: center; */
+	/* border-top: .1rem solid var(--text-color-2); */
+	/* padding: 1rem; */
+	/* padding-top: 2rem; */
```
Comment (Dev-AkshatGupta) [inline src/Components/Footer/Footer.css:37]: remove the commented code

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything looks cool to me in this you can merge it

File: src/Components/Footer/Footer.css [hunk 3]
```diff
@@ -29,12 +30,13 @@
 }
 
 .footer .credit {
-	font-size: 2rem;
-	text-align: center;
-	border-top: .1rem solid var(--text-color-2);
-	padding: 1rem;
-	padding-top: 2rem;
+	font-size: 1.5rem;
```
Comment (MuskanChhatrasal) [inline src/Components/Footer/Footer.css:33]: Remove commented code

File: src/Components/Footer/Footer.js [hunk 4]
```diff
@@ -1,23 +1,29 @@
...
+  return (
+    <section class="footer">
+      <div class="box-container">
+        <div class="box">
+          <a href="#">
+            {" "}
+            <i class="fab fa-twitter"></i> twitter{" "}
+          </a>
+          <a href="#">
+            {" "}
```
Comment (MuskanChhatrasal) [inline src/Components/Footer/Footer.js:13]: Remove these brackets

**Review by MuskanChhatrasal — APPROVED** [review]
Suggested few things just implement those. Else everything looks good to merge

**Comment by aayushjain-code** [comment]
Thanks for pr changes made and completed

**Pull request merged**